api/payments_test.go

```diff
@@ -254,9 +254,15 @@ func TestPaymentsRefundAmountTooHighOrLow(t *testing.T) {
 }
 
 func TestPaymentsRefundPaypal(t *testing.T) {
+	refundID := "4CF18861HF410323U"
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprint(w, `{"access_token":"EEwJ6tF9x5WCIZDYzyZGaz6Khbw7raYRIBV_WxVvgmsG","expires_in":100000}`)
+		if r.URL.Path == "/v1/oauth2/token" {
+			w.Header().Add("Content-Type", "application/json")
+			fmt.Fprint(w, `{"access_token":"EEwJ6tF9x5WCIZDYzyZGaz6Khbw7raYRIBV_WxVvgmsG","expires_in":100000}`)
+		} else if r.URL.Path == "/v1/payments/sale/"+secondTransaction.ProcessorID+"/refund" {
+			w.Header().Add("Content-Type", "application/json")
+			fmt.Fprint(w, `{"id":"`+refundID+`"}`)
+		}
```

Comment:
consider using a switch with a default case reporting for an unexpected value. Also consider using some counters to ensure these are called only once